src/enumerate.js

```diff
@@ -57,6 +57,11 @@ Sk.builtin.enumerate.prototype.next$ = function (self) {
     return self.tp$iternext();
 };
 
+Sk.builtin.enumerate.co_varnames = ["", "start"];
```

Comment:
lets call the first argument "iterable" 😄

Reply:
> Thank you! Looks good! One small comment.
Ok, I got it, I change the argument will get twice commit, 
Do I need to rebase the branch and make the commit to be once?

Reply:
> lets call the first argument "iterable"
Ok, I got it, I change the argument will get twice commit,
Do I need to rebase the branch and make the commit to be once?